dockerfiles/git.Dockerfile

```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
+
+ARG GO_VERSION=1.18
+ARG ALPINE_VERSION=3.16
+ARG GIT_VALIDATION_VERSION=v1.1.0
```

Comment:
Given that this ARG is only used in side the `gitvalidation` stage; perhaps consider moving it there. To reduce potential merge-conflicts if GO_VERSION or ALPINE_VERSION is to be updated and/or backported